### PR TITLE
Adjust to install OVNController CRD from ovn-operator instead of ovs-operator

### PR DESCRIPTION
**Pull request by booxter**
Also cleaned up now redundant targets for ovs-operator. The operator is retiring.

### CODE REVIEW

**Comment by booxter** [comment]
This should pass after https://github.com/openstack-k8s-operators/ovn-operator/pull/52 is merged.

File: Makefile [hunk 1]
```diff
@@ -91,20 +91,14 @@ OVNDBS              ?= config/samples/ovn_v1beta1_ovndbcluster.yaml
 OVNDBS_CR           ?= ${OPERATOR_BASE_DIR}/ovn-operator/${OVNDBS}
 OVNNORTHD           ?= config/samples/ovn_v1beta1_ovnnorthd.yaml
 OVNNORTHD_CR        ?= ${OPERATOR_BASE_DIR}/ovn-operator/${OVNNORTHD}
+OVNCONTROLLER       ?= config/samples/ovn_v1beta1_ovncontroller.yaml
+OVNCONTROLLER_CR    ?= ${OPERATOR_BASE_DIR}/ovn-operator/${OVNCONTROLLER}
+OVNCONTROLLER       ?= config/samples/ovn_v1beta1_ovncontroller.yaml
```
Comment (karelyatin) [inline Makefile:96]: Duplicated L94,95

**Comment by booxter** [comment]
CI failed with:

```
INFO[2023-05-22T13:45:00Z] Waiting for cluster claim openstack-k8s-operators-cluster-pool/9b282fcc-f7d6-46b1-9526-0cd90e6533b7 to be fulfilled. 
WARN[[20](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_install_yamls/284/pull-ci-openstack-k8s-operators-install_yamls-main-install-yamls-deploy/1660641433062739968#1:build-log.txt%3A20)23-05-22T14:45:00Z] Found 1 conditions for ClusterClaim:
  *[2023-05-22T13:45:00Z]NoClusters: No clusters in pool are ready to be claimed
```

I guess I should retest?

**Review by karelyatin — APPROVED** [review]
/lgtm

**Pull request merged**